wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallViewController/CallViewController.swift

```diff
@@ -281,7 +281,7 @@ final class CallViewController: UIViewController {
     private lazy var establishingCallStatusView = EstablishingCallStatusView()
 
     private func acceptDegradedCall() {
-        guard let userSession = ZMUserSession.shared() else { return }
+        guard let userSession = userSession as? ZMUserSession else { return }
```

Comment:
Can we make the voice channel compatible with `UserSession`?

Reply:
probably, but I'll do it in a next PR